### PR TITLE
1-Landing page 2-Framer animations 3- Header fixes 4-Routing

### PR DESCRIPTION
**Pull request by mihirverma7781**

### CODE REVIEW

File: src/Routes/Routes.js [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
 import { Switch, Route } from "react-router-dom";
 import Home from "../modules/Home";
-
+import Signup from "../modules/auth/signup";
+// import Login from "../modules/auth/Login";
 const Router = () => {
   return (
     <Switch>
       <Route exact path="/" component={Home} />
+      <Route exact path="/Signup" component={Signup} />
```
Comment (yash-sharma01) [inline src/Routes/Routes.js:10]: replace it with /signup

File: src/components/Header/index.js [hunk 2]
```diff
@@ -1,7 +1,17 @@
 import React, { useState } from "react";
 import Button from "../Button";
+import { useHistory } from "react-router-dom";
+
 const Header = () => {
   const [colorChange, setColorchange] = useState(false);
+  const path = window.location.pathname;
+  let extra;
+  if (path.includes("Signup") || path.includes("Login")) {
```
Comment (yash-sharma01) [inline src/components/Header/index.js:9]: same as above

**Review by yash-sharma01 — CHANGES_REQUESTED** [review]
make the following changes

File: src/components/Header/index.js (hunk 2 above)
Comment (mihirverma7781) [reply src/components/Header/index.js]: please check now

File: src/Routes/Routes.js (hunk 1 above)
Comment (mihirverma7781) [reply src/Routes/Routes.js]: please check now

**Review by yash-sharma01 — APPROVED** [review]

**Pull request merged**